src/components/tiltak/LagretTiltak.tsx

```diff
@@ -183,28 +178,29 @@ export const LagretTiltak = ({
             />
           )}
 
-          {!displayNyKommentar && !editererTiltak && (
+          {!displayNyKommentar && !editererTiltak && !vurdererTiltak && (
             <Row marginTop={"2rem"}>
-              {!readonly && !manglerVurderingFraLeder && (
-                <Button
-                  variant={"tertiary"}
-                  icon={<Edit aria-hidden />}
-                  onClick={() => setEditererTiltak(true)}
-                >
-                  Endre
-                </Button>
-              )}
-
-              {!readonly && kanSletteTiltak() && (
-                <SlettTiltakButton tiltakId={tiltak.tiltakId} />
-              )}
+              {!readonly &&
+                isTiltakCreatedByInnloggetRole() &&
+                (!isAudienceSykmeldt || !isVudert()) && (
```

Comment:
Med så mye audience-greier i denne filen hadde det kanskje vært lurt å vurdere å splitte filen i 2, eller alternativt ta inn children for å passe inn audience-relaterte greier.

Reply:
Ja, skal ta en titt hvordan det kan bli